src/components/multilineInput/styles/SplitMultilineInput.css

```diff
@@ -0,0 +1,22 @@
+.inputBlock {
+  margin-top: 0;
+}
+
+.inputLine {
+  display: flex;
+  justify-content: space-between;
+  position: relative;
```

Comment:
I'm wondering if we should use `className="flex space-between relative pb1"` instead defining the style here. I'm not against the css-modules, I'm just wondering what's the best in this case